src/java.base/share/classes/java/nio/charset/MalformedInputException.java

```diff
@@ -41,6 +41,9 @@
     @java.io.Serial
     private static final long serialVersionUID = -3438823399834806194L;
 
+    /**
+     * The length of the input byte sequence.
```

Comment:
Should this comment also refer to the character sequence?

Reply:
It could, maybe something like "The length of the input byte (or character) sequence." would work?

Reply:
or just "The length of the input" so that it is consistent with the description of the getInputLength method.

Reply:
That sounds good.

Reply:
Changed to "The length of the input." in the [most recent commit](https://github.com/openjdk/jdk/pull/3376/commits/9f36eec44dec7cd8f75f787c7d386e26d55e6826).